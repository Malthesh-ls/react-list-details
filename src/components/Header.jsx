const Header = ({ title }) => {
  return (
    <div className="header">
      <span>Welcome to {title}</span>
    </div>
  )
}

export default Header
