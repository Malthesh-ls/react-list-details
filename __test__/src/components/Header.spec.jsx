import renderer from 'react-test-renderer';
import Header from '../../../src/components/Header';

describe('Header Tests', () => {
  it('should match snapshot', () => {
    const snapshot = renderer
      .create(<Header title='test app' />)
      .toJSON();
    expect(snapshot).toMatchSnapshot();
  });
});