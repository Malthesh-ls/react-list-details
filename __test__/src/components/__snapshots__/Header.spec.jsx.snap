// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Tests should match snapshot 1`] = `
<div
  className="header"
>
  <span>
    Welcome to 
    test app
  </span>
</div>
`;
